src/pages/aurora-site/learning/vocabulary-content.jsx

```
@@ -1,69 +1,132 @@
+            </h2>
+            <Categories />
+          </div>
+        );
+      case "Saved Words":
+        <div
+          role="tabpanel"
+          id="saved-words-panel"
+          aria-labelledby="saved-words-tab"
+        >
+          <SavedWork setActiveTab={setActiveTab} />
+        </div>;
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)